packages/windbg.vm/tools/chocolateyinstall.ps1

```diff
@@ -5,16 +5,22 @@ try {
     $toolName = 'WinDbg'
     $category = 'Debuggers'
 
-    # It seems WinDbg is now distributed as an .appinstaller and we need to install it using Add-AppxPackage
-    Add-AppxPackage -AppInstallerFile 'https://windbg.download.prss.microsoft.com/dbazure/prod/1-0-0/windbg.appinstaller'
+    $exeUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-2402-24001-0/windbg.msixbundle"
+    $exeSha256 = "e941076cb4d7912d32a22ea87ad2693c01fa465227b4d1ead588283518de428f"
 
-    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
-    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-    $executableCmd  = Join-Path ${Env:WinDir} "system32\cmd.exe"
-    # Use `start` to close the open console
-    $executableArgs = "/C start WinDbgX.exe"
-    $executableDir  = Join-Path ${Env:UserProfile} "Desktop"
-    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executableCmd -Arguments $executableArgs -WorkingDirectory $executableDir -RunAsAdmin
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        url           = $exeUrl
+        checksum      = $exeSha256
+        checksumType  = "sha256"
+        fileFullPath  = Join-Path ${Env:TEMP} "$toolName.msixbundle"
+    }
+    Get-ChocolateyWebFile @packageArgs
+    Add-AppxPackage -Path $packageArgs.fileFullPath
+
+    $installDir = (Get-AppxPackage -Name "Microsoft.$toolName").InstallLocation
+    $executablePath = Join-Path $installDir "DbgX.Shell.exe"
```

Comment:
have you tested this code locally? I get an error when trying to run the shortcut:
![image](https://github.com/mandiant/VM-Packages/assets/16052290/1318c9bd-3e15-421c-aa7f-a087b882125a)

The following code works:
```suggestion
    $installDir = (Get-AppxPackage -Name "Microsoft.$toolName").InstallLocation
    $iconLocation  = Join-Path $installDir "DbgX.Shell.exe" -Resolve
    $executablePath = "$(where.exe WinDbgX.exe)"
    VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -iconLocation $iconLocation -RunAsAdmin
```

Note I have also added a `-Resolve` to `Join-Path` to ensure the `DbgX.Shell.exe` exists.

Reply:
Yes, I have tested locally strange...

Reply:
How, did you test ? I have tried from `Tools` directory and the Windows bar, it seems to work. Maybe it was the `RunAsAdmin` that was needed. Can you try again ?

> I have added the `-Resolve` parameter

Reply:
Weird, the only difference between our setup is that my empty VM only has `chocolatey`, I don't have a snapshot with `install.ps1` installed (yet). If I get some time, I will investigate on this, but anyway, your proposal seems to work for me too.

Reply:
It seems that the CI of windows-2019 doesn't appreciate `"$(where.exe WinDbgX.exe)"`.

Reply:
> Weird, the only difference between our setup is that my empty VM only has `chocolatey`, I don't have a snapshot with `install.ps1` installed (yet).

After creating a snapshot as [documented in the wiki](https://github.com/mandiant/VM-Packages/wiki/Local-testing) but with my current VM (which is build 22621) it still works (without the `where.exe`. So the problem, comes from Windows version.